projects/gnomad/deploy/gnomad-deployment-dev.yaml

```diff
@@ -0,0 +1,45 @@
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: gnomad-d-serve
+  labels:
+    state: serving
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: gnomad-d-serve
+  template:
+    metadata:
+      labels:
+        name: gnomad-d-serve
+    spec:
+      # volumes:
+      # - name: readviz
+      #   gcePersistentDisk:
+      #     pdName: gnomad-readviz-exons-gpd-3
+      #     fsType: ext4
+      #     readOnly: true
```

Comment:
I guess it's still kind of up in the air whether we want this in the dev server or not? So it seems reasonable to me to leave it here commented out.